src/libsystemd/sd-journal/journal-vacuum.c

```diff
@@ -48,6 +48,16 @@ static int vacuum_compare(const struct vacuum_info *a, const struct vacuum_info
         return strcmp(a->filename, b->filename);
 }
 
+static void vacuum_info_array_free(vacuum_info *list, size_t n) {
+        if (!list)
+                return;
+
+        for (vacuum_info *i = list; i < list + n; i++)
+                free(i->filename);
```

Comment:
You can reuse `FOREACH_ARRAY` (not a big matter tho)

Reply:
Indeed. I forgot about that one.